Berksfile.lock

```diff
@@ -123,7 +142,7 @@ GRAPH
   python (1.4.6)
     build-essential (>= 0.0.0)
     yum-epel (>= 0.0.0)
-  rabbitmq (3.3.0)
+  rabbitmq (3.9.0)
```

Comment:
It was using a function introduced in a newer version of Chef (or maybe it was a type attribute). Unfortunately, I don't recall what the function it was.